docs/guides/references/migration-guide.mdx

```diff
@@ -2,6 +2,90 @@
 title: Migration Guide
 ---
 
+## Migrating to Cypress 13.0
+
+This guide details the changes and how to change your code to migrate to Cypress
+version 13.0.
+[See the full changelog for version v13.0](/guides/references/changelog#13-0-0).
+
+### Cypress Cloud Test Replay
+
+[Test Replay](/guides/cloud/debugging/test-replay) is enabled by default in `v13.0` of the Cypress App.
+
+You may need to allowlist `capture.cypress.io` if you work with a strict VPN. See our FAQ section about [VPN subdomain allowlisting](/faq/questions/cloud-faq#Im-working-with-a-restrictive-VPN-Which-subdomains-do-I-have-to-allow-on-my-VPN-for-Cypress-Cloud-to-work-properly).
+
+With Test Replay enabled, the Cypress Runner UI is hidden by default when recording a run to the Cloud. If the Runner UI is needed during the run, you can enable it by passing [`--runner-ui`](/guides/guides/command-line#cypress-run-runner-ui) to the [`cypress run`](/guides/guides/command-line#cypress-run) command.
+
+You can [opt out](/guides/cloud/debugging/test-replay#Opt-out-of-Test-Replay) of this feature in Cloud project-level settings.
+
+### Video updates
+
+#### `video` is set to `false` by default
+
+You can continue recording video, by passing `video: true`, if you want video locally or you want video for some other reason, like in non-Chromium browsers where [Test Replay](/guides/cloud/debugging/test-replay) is not available.
```

Comment:
```suggestion
You can continue recording video by setting `video` to `true` either in your Cypress configuration or via [overriding options](/guides/references/configuration#Overriding-Options). This can be useful if you want video locally or want video for some other reason, like in non-Chromium browsers where [Test Replay](/guides/cloud/debugging/test-replay) is not available.
```